Assets/Scripts/KnopScript2.cs

```diff
@@ -0,0 +1,67 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class KnopScript2 : MonoBehaviour {
```

Comment:
Het mooist is als je deze zou kunnen mixen met KnopScript1,
misschien een array van GameObjecten (platform) maken en met een for each ze aanzetten.